app/controllers/services_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ServicesController < ApplicationController
+  def register
+    @service = Service.create
+    render json: @service, include: :models
```

Comment:
What happens if a transfer of data occurs from one microservice to another, but the app that obtains the data boots first? It'll attempt to register a model already registered by another service.

Or do we just consider that user error and not bother with handling that?